imap_processing/swe/l1a/swe_science.py

```diff
@@ -169,16 +173,30 @@ def swe_science(decom_data):
         ).output(),
     )
 
+    science_attrs = dataclasses.replace(
+        swe_cdf_attrs.l1a_science_attrs,
+        catdesc="Uncompressed counts from SWE",
+        fieldname="Uncompressed counts from SWE",
+        label_axis="Uncompressed counts from SWE",
+        units="counts",
+    )
     science_xarray = xr.DataArray(
         science_array,
         dims=["Epoch", "Energy", "Counts"],
-        attrs=swe_cdf_attrs.l1a_science_attrs.output(),
+        attrs=science_attrs.output(),
     )
 
+    raw_science_attrs = dataclasses.replace(
+        swe_cdf_attrs.l1a_science_attrs,
+        catdesc="Raw counts from SWE",
+        fieldname="Raw counts from SWE",
+        label_axis="Raw counts from SWE",
+        units="counts",
```

Comment:
I don't know how much I should be focusing on ISTP stuff at the moment, (my guess is not much?) but for the record I'd probably have:

catdesc="Raw electron counts at 12 energy bins between (xx) and (yy) eV"
fieldname="Raw Electron Counts"
label_axis="Counts"

Reply:
Whoops you merged right as I was writing lol

Reply:
sorry Bryan! Please add comment and I will open with new changes.